exporter/loadbalancingexporter/helpers.go

```diff
@@ -0,0 +1,85 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package loadbalancingexporter // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/loadbalancingexporter"
+
+import (
+	"go.opentelemetry.io/collector/pdata/pmetric"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+)
+
+// mergeTraces concatenates multiple ptrace.Traces into a single ptrace.Traces.
+func mergeTraces(traces ...ptrace.Traces) ptrace.Traces {
+	merged := ptrace.NewTraces()
+	empty := ptrace.Traces{}
+
+	for _, trace := range traces {
+
+		if trace == empty {
```

Comment:
Check num spans. Also you should not use zero-initialized pdata, per comments in the package that these instances are invalid.

Reply:
Got it, thank :)